notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>This is a great gif! From where did you get it? Also in a different notebook Thomas recommended linking to the 3Blue1Brown video about Fourier Transforms which I also love and could be applicable here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I got it from wikimedia commons, should I add a citation? I'll also link to this video. </p>

Reply:
<p>added credit and link </p>